src/images/index.d.ts

```diff
@@ -1,3 +1,8 @@
+declare module '*.jpg' {
```

Comment:
This file already had types for PNG and SVG images. This adds them for JPG images. In all cases, the type of the export is a string (the path to the file created by webpack).